tests/test_tcxparser.py

```diff
@@ -138,3 +154,10 @@ def test_no_error_if_no_position(self):
         tcx = TCXParser(tcx_file)
         self.assertEqual(tcx.latitude, None)
         self.assertEqual(tcx.longitude, None)
+
+    def test_hr_percent_in_zones_no_hr_data(self):
+        tcx_file = 'no_hr.tcx'
+        path = os.path.join(os.path.dirname(__file__), 'files', tcx_file)
+        tcx = TCXParser(path)
+        with self.assertRaises(NoHeartRateDataError):
+            tcx.hr_percent_in_zones({})
```

Comment:
Looks like travis is failing because of a flake8 complaint: `./tests/test_tcxparser.py:163:40: W292 no newline at end of file`. I'll go ahead and fix that now.

```suggestion
            tcx.hr_percent_in_zones({})

```